cmd/operator-sdk/main.go

```diff
@@ -87,29 +87,25 @@ func main() {
 	}
 }
 
-func checkDepManagerForCmd(cmd *cobra.Command) (err error) {
+func checkGoModulesForCmd(cmd *cobra.Command) (err error) {
 	// Certain commands are able to be run anywhere or handle this check
 	// differently in their CLI code.
 	if skipCheckForCmd(cmd) {
 		return nil
 	}
-	// Do not perform this check if the project is non-Go, as they will not have
-	// a (Go) dep manager.
+	// Do not perform this check if the project is non-Go, as they will be
+	// using go modules.
```

Comment:
This comment doesn't sound right. Non-Go projects won't use go modules will they? The old comment stated "as they will not have a (Go) dep manager" I think that comment is still valid.

Reply:
Good catch. Need a "not" in there.